Tests/ValetObjectiveCBridgeTests/VALValetTests.m

```diff
@@ -167,6 +172,271 @@ - (void)test_iCloudValetWithSharedAccessGroupIdentifier_accessibility_returnsNil
     XCTAssertNil(valet);
 }
 
+- (void)test_containsObjectForKey_returnsTrueWhenObjectExistsInKeychain;
```

Comment:
I realized while I was here that I didn’t have an explicit objective-c test for this wrapper method, so I wrote a couple